packaging/uploader.spec

```
@@ -32,6 +37,30 @@ a = Analysis(
         # Collected at the bundle root so paths.icon_file()'s frozen-case
         # lookup (bundle_dir() / "app.ico") finds it directly.
         (str(ICON), "."),
+        # Must reach the installed tree, not just the repository: the offer
+        # has to travel with the binaries it covers.
+        (str(ROOT / "THIRD-PARTY-NOTICES.md"), "."),
```

RuboCop (1.89.0)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)